lib/csvProcessor.js

```diff
@@ -98,6 +98,7 @@ CSVProcessor.prototype.individual = function() {
                 }
                 contextList.push(this.fileSpec);
                 if (errors[id].scope === 'hmda') {
+                    contextList.push(this.fileSpec.hmdaFile);
```

Comment:
It's not a stray change. It allows lookups for the CSV header for the dot notation of hmdaFile elements, like `transmittalSheet.property` or `loanApplicationRegister.property`.  This is the same type of thing that happens in the UI via the property label lookup filter.